src/pages/dho/Home.vue

```diff
@@ -292,36 +195,67 @@ export default {
         }
       }
       return stepIndex
-    }
-  },
-
-  methods: {
-    ...mapActions('proposals', ['saveDraft']),
-    getPaletteColor,
-    hidePlanBanner () {
-      localStorage.setItem('showPlanBanner', false)
-      this.isPlanBannerVisible = false
     },
 
-    hideWelcomeBanner () {
-      localStorage.setItem('showWelcomeBanner', false)
-      this.isWelcomeBannerVisible = false
-    },
+    isUpVoteElectionBannerVisible () { return this.upvoteElection.isActive },
 
-    hideSignUpElectionBanner () {
-      localStorage.setItem('showSignUpElectionBanner', false)
-      this.isSignUpElectionBanner = false
-    },
+    isWelcomeBannerVisible () { return true },
 
-    votingTimeLeft () {
-      const end = this.upvoteElectionData.upcomingElection?.length ? new Date(this.upvoteElectionData.startTime) : new Date(this.upvoteElectionData.endTime)
-      const now = Date.now()
-      const t = end - now
-      if (t < 0) {
-        this.$apollo.queries.upvoteElectionQuery.refetch()
+    upvoteElectionBanner () {
+      return {
+        title: clsx({
+          'Sign up for the election!': this.currentStepIndex === 0,
+          'The election is on progress': this.currentStepIndex > 0 && this.currentStepIndex < 4,
+          'The election in completed!': this.currentStepIndex === 4
+        }),
+        description: clsx({
+          'Hello Community members! We are soon running our Upvote Election! It will allow everyone in the AwesomeDAO community to actively participate to decision making and building our cool project together! How does it work? In a nutshell: we will run community proposals that can be voted by delegates badge holders. If you feel like being a delegate, apply now for a badge! If want to just vote your favourite delegates, apply for a voter bade!': this.currentStepIndex === 0,
```

Comment:
@arsenijesavic I don't really like this, think that we should implement new features with i18n by default.